src/SFML/Window/GlContext.cpp

```diff
@@ -388,7 +388,7 @@ struct GlContext::TransientContext
     ////////////////////////////////////////////////////////////
     static std::optional<TransientContext>& get()
     {
-        thread_local std::optional<TransientContext> transientContext;
+        thread_local std::optional<TransientContext> transientContext; // NOLINT(misc-use-internal-linkage)
```

Comment:
I assumed this was a false positive.

Reply:
Could also just add `static` in front of it even though `thread_local` already implies this according to the standard.